guides/common/modules/proc_importing-a-repository.adoc

```diff
@@ -0,0 +1,45 @@
+[id="Importing_a_Repository_{context}"]
+= Importing a Repository
+
+You can use the archive that the `hammer content-export` command outputs to import a repository in another organization.
+For more information about exporting content of a repository, see xref:Exporting_a_Repository_{context}[].
+
+.Prerequisites
+
+To import a repository to an Organization's library lifecycle environment ensure that {ProjectServer} on which you want to import, meets the following conditions:
+
+* The export files must be in a directory under `/var/lib/pulp/imports`.
+* The directory must have `pulp:pulp` ownership so that Pulp can read and write the `.json` files in that directory.
+* If the export contains any Red Hat repositories, the manifest of the importing organization must contain subscriptions for the products contained within the export.
+* The user importing the content must have the 'Content Importer' Role.
```

Comment:
```suggestion
* The user importing the content must have the _Content Importer_ Role.
```

Maybe either italic or code? I don't think we currently use quotes to highlight text.